README.md

```diff
@@ -15,16 +15,16 @@ ringOS source code
 [**ringOSY stable release**](https://github.com/ringwormGO-organization/ringOS/tree/ringOSY)
 
 ### Contributing and issues
-If you have problem or idea create **issue**.
+If you have problem or idea create an **issue**.
 
-If you want contribute in this project create **Pull Request**.
+If you want contribute to this project create a **Pull Request**.
 
 # About ringOS
-ringOS is ringwormGO operating system based on PonchoOS with supprot UEFI boot mode only.
+ringOS is ringwormGO operating system based on PonchoOS with support for UEFI boot modes only.
```

Comment:
There is only BIOS or UEFI boot mode

Reply:
What I've written clarifies that - the language you used originally misspelled "support" and requires the adjective "for" to be gramatically correct. What I've written says the same thing as your original statement but is considered gramatically correct in english

Reply:
That is correct, but stil there is one UEFI boot mode, we can use `modes` if we think on UEFI revisions

Reply:
I removed the "s" in "modes" to make it singular

Reply:
Good